coriolisclient/cli/endpoint_storage.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2017 Cloudbase Solutions Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Command-line interface sub-commands related to endpoints.
+"""
+from cliff import lister
+from coriolisclient.cli import formatter
+
+
+class EndpointStorageFormatter(formatter.EntityFormatter):
+
+    columns = ("ID",
+               "Name",
+               )
```

Comment:
This is unnecessary, please put the closing paren on the previous line (and remove the tralining `,`)